lib/LDA.py

```diff
@@ -1,33 +1,55 @@
 #!/usr/bin/env python
-import numpy
+"""
+A module that contains the content-based recommender LDARecommender that uses
+LDA.
+"""
 from lib.content_based import ContentBased
 from sklearn.decomposition import LatentDirichletAllocation
 from sklearn.feature_extraction.text import CountVectorizer
 
 
 class LDARecommender(ContentBased):
+    """
+    LDA Recommender, a content based recommender that uses LDA.
+    """
     def __init__(self, abstracts, evaluator, config, verbose=False):
+        """
+        Constructor of ContentBased processor.
+        @param (list[str]) abstracts: List of the texts of the abstracts of the papers.
+        @param evaluator: An evaluator object.
```

Comment:
Let's assume that the type here is evaluator. so let's make it "@param (evaluator) evaluator: An evaluator object." or "@param (object) evaluator: An evaluator object." I know it seems redundant but we will need it for the docs generation